prowler/providers/aws/services/backup/backup_vaults_exist/backup_vaults_exist.py

```diff
@@ -0,0 +1,27 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.backup.backup_client import backup_client
+
+
+class backup_vaults_exist(Check):
+    def execute(self):
+        findings = []
+
+        for backup_vault in backup_client.backup_vaults:
```

Comment:
Please, change here the logic like in the checks `backup_plans_exist` and `backup_reportplans_exist`